source/apiVolontaria/volunteer/serializers.py

```diff
@@ -414,11 +415,39 @@ class ParticipationBasicSerializer(serializers.ModelSerializer):
 
     # Explicitly declare the BooleanField to make it "required"
     standby = serializers.BooleanField()
+    invit_icalandar = serializers.BooleanField(
+        default=False,
+        write_only=True,
+    )
     user = UserBasicSerializer(
         read_only=True,
         default=serializers.CurrentUserDefault(),
     )
 
+    def create(self, validated_data):
+        participation = models.Participation()
+        participation.standby = validated_data['standby']
+        participation.event = validated_data['event']
+        participation.user = validated_data['user']
+
+        if 'invit_icalandar' in validated_data.keys():
+            participation.presence_duration_minutes = validated_data['invit_icalandar']
```

Comment:
Why did you change the duration minute of the participation in function of the fact we want an 'ics' file or not ? I'm lost here.

Reply:
done